mkdocs.yml

```
@@ -61,9 +61,10 @@ plugins:
           de: Deutsch
           fr: Français
           es: Español
-          jp: 日本語
-          zh_CN: 简体中文
+          ja: 日本語
+          zh: 简体中文
           sv: Svenska
+          pt: Portuguese
```

```suggestion
          pt: Português
```

We've been using the native term for the languages, other than that LGTM.